contracts/clarity-bitcoin.clar

```diff
@@ -374,5 +360,5 @@
 ;; This function must only called with the merkle root of the provided header
 (define-private (was-tx-mined-internal (height uint) (tx (buff 1024)) (header (buff 80)) (merkle-root (buff 32)) (proof { tx-index: uint, hashes: (list 14 (buff 32)), tree-depth: uint}))
     (if (verify-block-header header height)
-        (verify-merkle-proof (get-reversed-txid tx) (reverse-buff32 merkle-root) proof)
+        (verify-merkle-proof (SHA-256d tx) (reverse-buff32 merkle-root) proof)
```

Comment:
From a user point of view, it is much more difficult to understand why we have `sha256d` here.